test/benchmark/load_worlds.py

```diff
@@ -0,0 +1,24 @@
+def run_load_worlds_benchmark():
+    """List worlds and their load time.
+    Note that any first-time imports will be attributed to that world, as it is cached afterwards.
+    Likely best used with isolated worlds to measure their time alone."""
+    import logging
+
+    from Utils import init_logging
+
+    # get some general imports cached, to prevent it from being attributed to one world.
+    import BaseClasses, Launcher, Fill
```

Comment:
```suggestion
    import orjson
    orjson.loads("{}")  # orjson runs initialization on first use

    import BaseClasses, Launcher, Fill
```
Not sure if there are other things we should import/init as well.